qa/zcash/smoke_tests.py

```diff
@@ -218,47 +218,49 @@ def async_txid_cmd(results, case, zcash, name, args=[]):
             results[case] = False
     return txid
 
```

Comment:
Comment on the `print` at lines 123-127: since there's no guarantee that the arguments are printed in a way compatible with invoking `zcash-cli` at a shell, and since `zcash-cli` is not actually used by this script, we should remove "$ zcash-cli" from the print output and just print the RPC call. I would also not try to hide that the arguments are represented as a Python array. Something like this:
```python
    print('%s: %s %r' % (case.ljust(3), name, args))
```